backend/src/common/enums/experience.enum.ts

```diff
@@ -3,4 +3,11 @@ export enum Experience {
   EXPERIENCED,
   CHIEF_EXPERIENCED,
   OUTSIDE_EXPERIENCED,
-}
\ No newline at end of file
+}
+
+export enum ExperienceName {
+  INTERESTED = 'INTERESTED',
```

Comment:
Going back on what I said on my PR, can you do this `INTERESTED = 'INTERESTED'` in the regular enums, while enumName is meant for user friendly names like `INTERESTED = 'Interested'`?

I was wrong before about the blank enum
`{
INTERESTED,
}`

being the same as definition
`{
INTERESTED = 'INTERESTED',
}`

Reply:
For sure! I am going to update this on TC-85 if that's okay? (it's pointed to this branch)